pyop2/compilation.py

```diff
@@ -31,14 +31,12 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from __future__ import absolute_import, print_function, division
-import six
-from six.moves import input
-
+from __future__ import absolute_import
```

Comment:
I did the fork yesterday after our call so I'm not sure what happened there. I'll try a rebase to see what gives.